### ISSUE TITLE
fix Codesandbox call stack size and Lato-Black.svg

### ISSUE THREAD

**Issue by ghost** [nexxtway/react-rainbow#1909]
react-rainbow-component v1.18.1
react v16.8.6

Codesandbox console: RangeError: Maximum call stack size exceeded
![Screenshot 2020-10-12 at 8 06 10 AM](https://user-images.githubusercontent.com/67747427/95707382-abdde200-0c59-11eb-906f-70b92b3f81bc.png)

**Comment by LeandroTorresSicilia** [comment]
@organization983 can you provide more context? Like the code, you used in the codexandbox

**Comment by wildergd** [comment]
problem is related to embed assets for legacy styles... (same problem happened when the library within a Next.js project).. is not exactly a bug... however, one solution is to move legacy styles to a cdn and add the corresponding <link ....> to head when Application is mount

**Comment by LeandroTorresSicilia** [comment]
@ghost this has been addressed in version 1.24.1, when using Codesandbox I recommend you to import the components direct like:
```
import Application from 'react-rainbow-components/components/Application';
```
since the other way:
```
import { Application } from 'react-rainbow-components';
```
require all components and some of them still have problems with SSR
